src/App.jsx

```diff
@@ -1,56 +1,128 @@
-import { initialColors } from "./lib/colors";
+import { initialThemes, initialColors } from "./lib/colors";
 import { uid } from "uid";
 import useLocalStorageState from "use-local-storage-state";
-import Color from "./Components/Color/Color";
-import ColorForm from "./Components/ColorForm/ColorForm";
+import Themes from "./Components/Themes/Themes";
 import "./App.css";
+import { ThemesMenu } from "./Components/ThemesMenu/ThemesMenu";
+import { useState } from "react";
 
 function App() {
+  const [themes, setThemes] = useLocalStorageState("themes", {
+    defaultValue: initialThemes,
+  });
+
+  const [currentTheme, setCurrentTheme] = useState(themes[0]);
+
   const [colors, setColors] = useLocalStorageState("colors", {
     defaultValue: initialColors,
   });
 
```

Comment:
you can  now remove colors state and use for example :`const [isAddingTheme, setIsAddingTheme] = useState(false);`